oxlate/response.py

```diff
@@ -0,0 +1,63 @@
+import json
+import re
+
+from .headers import Headers
+from .cookie import Cookie
+
+class Response:
+    def __init__(self, status=200, content_type='text/plain', body=None):
+        self.set_status(status)
+        self.set_content_type(content_type)
+        self.set_body(body)
+        self._cookies = []
+        self._headers = Headers()
+
+    def set_status(self, value):
+        self._status = value
+        return self
+
+    def set_content_type(self, content_type):
+        self._content_type = content_type
+        return self
+
+    def set_content_type_json(self):
+        self._content_type = "application/json"
+        return self
+
+    def set_content_type_html(self):
+        self._content_type = "text/html"
+        return self
+
+    def set_body(self, body):
+        self._body = body
+        return self
+
+    def add_cookie(self, name, value=None, expires_at=None, path=None, domain=None):
```

Comment:
As above, perhaps add some sort of `expires_immediately=True` option?

Reply:
added `delete_cookie`